test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -114,11 +114,11 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>Using JVM : ${sdk.executable}${sdk.executable.ext}</echo>
 		<echo>classname = "j9vm.test.corehelper.StackMapCoreGenerator"</echo>
 		<echo>Java VM Args:</echo>
-		<echo>	jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset,modified=mod1</echo>
+		<echo>	jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset -Xitsn1000</echo>
```

Comment:
Why is `-Xitsn1000` added?

Reply:
we are running `-Xitsn1000` (see line 122). 

I added it into the `echo` so it prints out the exact jvm options that we use in the console output.